extensions/geode-modules/src/main/java/org/apache/geode/modules/session/catalina/DeltaSession.java

```diff
@@ -616,7 +642,7 @@ public int getSizeInBytes() {
     return size;
   }
 
-  @SuppressWarnings({"unchecked", "rawtypes"})
+  @SuppressWarnings({"unchecked"})
```

Comment:
While we're here, it might be worth cleaning up uses of this warning suppression using the `uncheckedCast()` method, as used elsewhere in this change set. There is one cast in this method (line 651) and one on line 633 of `getSizeInBytes()`.